ui_framework/components/_common_styles.scss

```diff
@@ -15,3 +15,10 @@
 body {
   font-family: $globalFontFamily;
 }
+
+/**
+ * 1. Required for IE11.
+ */
+main {
+  display: block; /* 1 */
+}
```

Comment:
@cjcenizal I ran both of the tasks, none of them touches git managed files for me?